Source/Core/Core/HW/EXI_DeviceSlippi.h

```diff
@@ -237,9 +245,13 @@ class CEXISlippi : public IEXIDevice
 	std::unique_ptr<Slippi::SlippiGame> m_current_game = nullptr;
 	SlippiSpectateServer *m_slippiserver = nullptr;
 	SlippiMatchmaking::MatchSearchSettings lastSearch;
+	SlippiMatchmaking::MatchmakeResult recentMmResult;
```

Comment:
i'd really prefer if we used snake case (with the appropriate prefix) for all new vars, but that's just cause i converted everything to snake case for mainline to match the style guide. not blocking this PR for that alone.